hw/opentitan/ot_usbdev.c

```diff
@@ -632,6 +959,24 @@ static void ot_usbdev_set_vbus_gate(OtUsbdevState *s, bool gate)
     ot_usbdev_update_vbus(s);
 }
 
+/*
+ * Retire any IN packet on the given endpoint.
+ *
+ * Mark any ready packet as pending but not ready and not sending.
+ */
+static void ot_usbdev_retire_in_packets(OtUsbdevState *s, uint8_t ep)
+{
+    uint32_t configin = s->regs[R_CONFIGIN_0 + ep];
```

Comment:
```suggestion
    g_assert(ep < USBDEV_PARAM_N_ENDPOINTS);
    uint32_t configin = s->regs[R_CONFIGIN_0 + ep];
```
(repeated in a few functions).